uid2_client/encryption.py

```diff
@@ -427,17 +544,32 @@ class DecryptedToken:
     """Details extracted from a decrypted advertising token.
 
     Attrs:
-        uid2 (str): universal ID string
+        uid (str): universal ID string
         established (datetime): UTC date/time for when the token was first generated
         site_id (int): site ID which the token is originating from
         site_key_site_id (int): site ID of the site key which the token is encrypted with
     """
 
-    def __init__(self, uid2, established, site_id, site_key_site_id):
-        self.uid2 = uid2
```

Comment:
This makes this release a breaking change, so we need to bump the major version.

Reply:
Thanks for pointing this out. I was trying not to break any existing functionality, for now I added a property alias `uid2` for existing clients to work. At the same time the overall changes in the PR look big enough to me to warrant a major version upgrade.